reference/process-beans.md

```diff
@@ -3,6 +3,28 @@
 Process beans are Spring beans that can be used inside a Camunda BPMN process. The BPMN expression fields have access to
 all process beans. This page gives an overview of all available process beans.
 
+## ConnectorService
+
+This process bean makes it possible to interact with connectors inside the BPMN process.
+
+```kotlin
+fun getConnectorTypes()
+fun getConnectorInstances(pageable: Pageable = Pageable.unpaged())
+fun getConnectorInstancesByType(typeId: UUID, pageable: Pageable = Pageable.unpaged())
+fun getConnectorInstancesByTypeName(typeName: String, pageable: Pageable = Pageable.unpaged())
```

Comment:
I guess this is a little bit late, but shouldn't this either:
1. Consist of a single table  with the method in 1 column and the description in the second colum
2. Just be the list from line 20 to 26.

Right now you're listing the same thing twice for the most part.

Reply:
I changed it. But not to a table because that made the functions look bad